Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -90,17 +101,10 @@ final class PledgeDescriptionCell: UITableViewCell, ValueCell {
     _ = self.descriptionLabel
       |> descriptionLabelStyle
 
-    _ = self.learnMoreLabel
+    _ = self.learnMoreButton
       |> checkoutBackgroundStyle
-    _ = self.learnMoreLabel
-      |> learnMoreLabelStyle
-  }
-
-  // MARK: - Configuration
-
-  func configureWith(value: String) {
-    _ = self.dateLabel
-      |> \.text .~ value
+    _ = self.learnMoreButton
```

Comment:
Are these styles split up for compiler reasons? Or can we do:

```
_ = self.learnMoreButton
  |> checkoutBackgroundStyle
  |> learnMoreButtonStyle
```

Reply:
These are split because the compiler gives us an error. This is because `learnMoreButtonStyle` is expecting a `UITextView` but `checkoutBackgroundStyle` is returning a `UIView`. 
Written like this it would work: 
`_ = self.learnMoreButton
  |> learnMoreButtonStyle
  |> checkoutBackgroundStyle
` 
 Let me know if I should make that change.

Reply:
We also split it because the first part sets the background and the next the style. That way we don't have to worry about the order of passing the `UITextView`.